tests/test_loadflow.py

```diff
@@ -163,6 +164,8 @@ def test_run_lf_with_report():
     pp.loadflow.run_ac(n, reporter = reporter)
     report2 = str(reporter)
     assert len(report2) > len(report1)
+    json_report = reporter.json
+    json.loads(json_report)
```

Comment:
is it possible to have some basic assertion on the content ? (I don't know what, maybe just that it's not empty ...)

Reply:
done: added an assertion on json_report'content